From d7d910cf84b7ec295aae92dd0172090cc1d65eac Mon Sep 17 00:00:00 2001
From: Nicholas Devenish <ndevenish@gmail.com>
Date: Sat, 28 Dec 2024 17:50:09 +0000
Subject: [PATCH 6/6] Remove cadquery

The actual dependency is OCP, via cadquery-ocp, but on
conda-forge the ocp package installs a python library
but doesn't install the metadata properly. Remove here
so that pip-check passes.
---
 pyproject.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 83056ab..f636cf8 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -35,7 +35,6 @@ classifiers = [
 ]
 
 dependencies = [
-    "cadquery-ocp >= 7.7.0",
     "typing_extensions >= 4.6.0, <5",
     "numpy >= 1.24.1, <2",
     "svgpathtools >= 1.5.1, <2",
-- 
2.46.2

